css/styles.css

```diff
@@ -0,0 +1,95 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* Globals */
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+span {
+  border: 1px solid gray;
+  width: 50px;
+  padding: 10px 12px;
+}
+
+/* Head Area CSS */
+.header {
+  padding: 12px 24px 12px 24px;
+  background-color: #3c3a39;
+}
+
+.developer {
+  font-family: 'Roboto', sans-serif;
+}
+
+.header ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-content: center;
+  padding: 4px;
+}
+
+.header ul a {
+  font-size: 18px;
+  color: #fff;
+  text-decoration: none;
+}
+
+/* Welcome Area CSS */
+.welcome-area {
+  display: flex;
+  flex-direction: column;
+  background-image: url('/images/header-bg.svg');
```

Comment:
Good job for putting the illustration image into the CSS `background-property`, but this is currently not able to load on the browser, please make sure to fix this.

![image](https://user-images.githubusercontent.com/34779928/191607538-eb5bdadc-1c79-40ef-8838-6de280587129.png)

Reply:
@aimalamiri, I am able to see the background Image both on chrome and Safari browsers.
I do not understand why you are not able to see the background image

![Screenshot 2022-09-23 at 19 23 13](https://user-images.githubusercontent.com/78411141/192007486-a479d280-ea50-45cb-914b-458d1f227941.png)


![Uploading Screenshot 2022-09-23 at 19.23.49.png…]()